copter/source/docs/autotune.rst

```diff
@@ -107,6 +139,7 @@ Additional Notes
 -   AutoTune is sometimes unable to find a good tune for frames with very soft vibration dampening of the autopilot or very flexible arms.
 -   For best results the copter shouldn't be allowed to build up too much horizontal speed. This can be prevented by applying a quick correction between tests (twitches) to stop the vehicle from flying too fast.
 -   Be advised that AutoTune will engage from Stabilize, so don't accidentally flip your AutoTune switch until you are in AltHold and ready to begin the procedure.
+-   As a general rule, for Pitch and Roll, P and I should be equal, and D should be 1/10th P. For Yaw, I should be 1/10th P and D = 0, in most cases.
```

Comment:
I'm not sure we should add this new line, "As a general rule, for Pitch and Roll, P and I should be equal, and D should be 1/10th P" because although it's a good rule for manual tuning, I don't think AutoTune will necessarily come up with this ratio of P/I to D.

Reply:
it does...I think...
#define AUTOTUNE_PI_RATIO_FINAL            1.0f     // I is set 1x P after testing
#define AUTOTUNE_YAW_PI_RATIO_FINAL        0.1f     // I is set 1x P after testing
 and during the test
attitude_control->get_rate_yaw_pid().kD(0.0f);

Reply:
Ah, right.  Just the "D should be 1/10th P" is not correct I'm pretty sure.

Reply:
ah, right...that's for manual tuning..will remove...autotune actually has a D phase for R/P...